pkgs/tools/wayland/wluma/default.nix

```diff
@@ -24,25 +24,14 @@ rustPlatform.buildRustPackage rec {
   nativeBuildInputs = [
     makeWrapper
     pkg-config
+    rustPlatform.rustBindgenHook
   ];
 
   buildInputs = [
     udev
-    v4l-utils.lib
+    v4l-utils
   ];
 
-  LIBCLANG_PATH="${llvmPackages.libclang.lib}/lib";
```

Comment:
I think there was also one in firefox. But I could also understand if you don't want to tackle this one yet - but it would be a good test.

Reply:
I added a commit for firefox. it builds.